src/crypto.ts

```diff
@@ -87,9 +86,8 @@ export function genKeyPairFromSeed(seed: string): KeyPair {
   validateString("seed", seed, "parameter");
 
   // Get a 32-byte key.
-  const derivedKey = misc.pbkdf2(seed, "", 1000, 32 * 8);
-  const derivedKeyHex = codec.hex.fromBits(derivedKey);
-  const { publicKey, secretKey } = sign.keyPair.fromSeed(hexToUint8Array(derivedKeyHex));
+  const derivedKey = pbkdf2Sync(seed, "", 1000, 32, "sha256");
```

Comment:
as for sha256 I checked jscl and that was the default function we used so that's fine but I'm just concerned about the change in key length here - used to be key length 32 * 8 and now it's just 32

Reply:
My best guess is one library uses bits and another uses bytes? `crypto.test.ts` checks the lengths and checks against a hard-coded key derivation from a given input, and these are the parameters that got the same output from `sign.keyPair.fromSeed()`, so I'm pretty confident they're correct.